bitnami/mariadb-galera/templates/statefulset.yaml

```diff
@@ -213,6 +213,9 @@ spec:
                 - bash
                 - -ec
                 - |
+                  if [[ -f "${MARIADB_ROOT_PASSWORD_FILE:-}" ]]; then
+                      MARIADB_ROOT_PASSWORD=$(cat "$MARIADB_ROOT_PASSWORD_FILE")
+                  fi
                   exec mysqladmin status -u$MARIADB_ROOT_USER -p$MARIADB_ROOT_PASSWORD
```

Comment:
Could you please use the same syntax as the Readiness and Startup probes?

```
                  password_aux="${MARIADB_ROOT_PASSWORD:-}"
                  if [[ -f "${MARIADB_ROOT_PASSWORD_FILE:-}" ]]; then
                      password_aux=$(cat "$MARIADB_ROOT_PASSWORD_FILE")
                  fi
                  exec mysqladmin status -u"${MARIADB_ROOT_USER}" -p"${password_aux}"
```

I'd prefer having all 3 of them using the same syntax if possible.

Reply:
Hi @migruiz4 ,

My pleasure. I spent hours trying to figure out why my pods were restarting so thought I should contribute here as well. :)
I have made the required changes as per your suggestions in the PR. Please take a look.